server/src/db/queries/carsQueries/getCarsQuery.ts

```diff
@@ -0,0 +1,45 @@
+import { Op } from 'sequelize';
+import { Car } from '../../models';
+
+const getCarsQuery = async ({
```

Comment:
let us add `state` and `page` as search params, and return matching results count using `findAndCountAll`,
Also I think No need to trim values here,
And maybe we can validate `req.query`